crates/core/flags/hiargs.rs

```diff
@@ -65,6 +65,7 @@ pub(crate) struct HiArgs {
     line_number: bool,
     max_columns: Option<u64>,
     max_columns_preview: bool,
+    max_columns_preview_before: Option<u64>,
```

Comment:
We can combine this to `max_columns_preview: Option<u64>`, but will cause breaking changes. Is this acceptable?